src/app/api/clients/[id]/route.ts

```
@@ -0,0 +1,60 @@
+import { NextRequest, NextResponse } from 'next/server';
+import { prisma } from '@/lib/db';
+
+import { NextRequest, NextResponse } from 'next/server';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'NextRequest'. Consider to delete it or rename it.

'NextRequest' is defined here:


(lint/suspicious/noRedeclare)